torch/csrc/jit/script/module.cpp

```diff
@@ -368,6 +369,37 @@ void Module::dump(
             << std::endl;
 }
 
+namespace {
+void export_opnames(const script::Module& m, std::unordered_set<std::string>& opnames) {
+  auto methods = m.get_methods();
+  for (const auto& method : methods) {
```

Comment:
Nit: `for (const auto& method : m.get_methods()) {`, no need in having `methods` variable.